governance/polls/Inclusion Poll - Governance Core Unit MIP Set - March 8, 2021.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Inclusion Poll for Governance Core Unit (GOV-001) MIP Set - March 8, 2021
+summary: Signal your support or opposition for this proposal being included in this month's governance cycle. 
+discussion_link: https://forum.makerdao.com/t/mip39c2-sp3-governance-core-unit-gov-001/6348
+vote_type: Plurality Voting
+categories:
+   - Governance
+   - MIPs
+   - Core Unit
+   - Budget
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Inclusion Poll for Governance Core Unit (GOV-001) MIP Set - March 8, 2021
+
+The Governance Facilitators have placed an inclusion poll into the [voting system](https://vote.makerdao.com/polling) as part of their responsibilities as defined in [MIP3](https://github.com/makerdao/mips/blob/master/MIP3/mip3.md). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, March 8 at 16:00 UTC.
+
+**This is a binary vote. In a binary vote, you may vote for a single option. You should vote for the option which you prefer. Alternatively, if you would accept either of the two options and have no preference between them, you should abstain.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to this proposal being included in this month's governance cycle.
+
+This Inclusion Poll MIP Set contains the following MIPs:
+* [MIP39c2-SP3: Governance Core Unit - GOV-001](https://forum.makerdao.com/t/mip39c2-sp3-governance-core-unit-gov-001/6348)
+* [MIP40c2-SP3: Core Unit Budget - GOV-001](https://forum.makerdao.com/t/mip40c2-sp3-core-unit-budget-gov-001/6349)
+* [MIP41c4-SP3: Core Unit Facilitator Onboarding - GOV-001](https://forum.makerdao.com/t/mip41c4-sp3-core-unit-facilitator-onboarding/6350)
+
+Please review the discussion [thread](https://forum.makerdao.com/t/mip39c2-sp3-governance-core-unit-gov-001/6348) to inform your position on this proposal before voting. It was also mentioned and possibly discussed on the [Governance and Risk call](https://forum.makerdao.com/t/agenda-discussion-scientific-governance-and-risk-131-thursday-march-4-17-00-utc/6737) on March 4, 2021. 
+
+## Outcomes
+
+This poll implements an **Inclusion Threshold.** The Inclusion Threshold is currently set to **3,000 MKR**.
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option by at least 3,000 MKR then the following actions will be taken:**
+* The Governance Facilitator(s) will confirm the inclusion of this proposal within the next Monthly Governance Poll on the Governance and Risk call on March 11, 2021. 
+* The Monthly Governance Poll occurs in week 3 of the [Governance Cycle](https://github.com/makerdao/mips/blob/master/MIP3/mip3.md).
+
+Note: This set of subproposals will only be included if the Core Unit MIP Set is included as well.
```

Comment:
I would add this to the bullet points above as part of the outcome.
"These proposals will only continue to the Monthly Governance Poll if the Core Unit MIP Set is also included." Something like that.

Reply:
Probably needs changing for the other dependent MIP Sets.